pandas/io/parsers.py

```diff
@@ -209,6 +209,11 @@
 warn_bad_lines : boolean, default True
     If error_bad_lines is False, and warn_bad_lines is True, a warning for each
     "bad line" will be output. (Only valid with C parser).
+delim_whitespace : boolean, default False
+    Specifies whether or not whitespace (e.g. ``' '`` or ``'\t'``) will be used
+    as the delimiter. Equivalent to ``'\+s'`` in regex. If this option is set
```

Comment:
say (both her and in the docs), that ``delim_whitespace=True`` equiv to ``sep='\s+'`` (not just that its the same regex)